tests/InstrEngineTests/NaglerInstrumentationMethod/NaglerInstrumentationMethod.cpp

```diff
@@ -14,6 +14,14 @@ const WCHAR CInstrumentationMethod::TestScriptFileEnvName[] = L"Nagler_TestScrip
 const WCHAR CInstrumentationMethod::TestScriptFolder[] = L"TestScripts";
 const WCHAR CInstrumentationMethod::IsRejitEnvName[] = L"Nagler_IsRejit";
 
+ILOpcodeInfo ilOpcodeInfo[] =
+{
+#define OPDEF(ord, code, name,  opcodeLen, operandLen, type, alt, flags, pop, push) \
+    { name, (DWORD)opcodeLen, (DWORD)operandLen, ##type, alt, flags, (DWORD)pop, (DWORD)push},
+#include "ILOpcodes.h"
+#undef OPDEF
+};
+
```

Comment:
What were the reasons for moving this from NaglerInstrumentMethodEntry.h?

Reply:
This was moved here since `dllmain.cpp` will indirectly pull in `NaglerInstrumentMethodEntry.h`, resulting in an error regarding `ilOpCodeInfo` already being defined. I'm open to alternative solutions to this if you have an idea.